src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -823,54 +861,55 @@ public static Text of(Object... objects) {
         ClickAction<?> clickAction = null;
         ShiftClickAction<?> shiftClickAction = null;
         boolean changedFormat = false;
+        boolean appendTexts = false;
```

Comment:
If this/`APPEND_TEXTS` is for https://github.com/SpongePowered/SpongeAPI/issues/1005, I'd default to true.